bangazon-clumsy-penguins/BangazonAPI
# Pp product type

# Product-Type

## Description
Product-Type Controller with Basic CRUD functionality to interact with Product-Type table

## Steps to Test

start API server then run GET, POST, PUT, DELETE.
Get should return a list of all product types, or a single product type when an Id is provided

##### System configuration.

Windows 10 

##### 3rd party libraries that need to be installed.

none

##### Command line utilities to run.

none

##### Link to Feature Ticket
#5
